cli/pcluster/pcluster.py

```diff
@@ -577,7 +577,45 @@ def instances(args):
         LOGGER.info("Run 'awsbhosts --cluster %s' to list the compute instances", args.cluster_name)
 
 
-def command(args, extra_args):
+def _get_master_server_ip(stack_name, config):
+    """
+    Get the IP Address of the MasterServer
+
+    :param stack_name: The name of the cloudformation stack
+    :param config: Config object
+    :return private/public ip address
+    """
+    ec2 = boto3.client(
+        "ec2",
+        region_name=config.region,
+        aws_access_key_id=config.aws_access_key_id,
+        aws_secret_access_key=config.aws_secret_access_key,
+    )
+
+    master_id = get_master_server_id(stack_name, config)
+    if master_id is []:
+        LOGGER.info("MasterServer not running. Can't SSH")
+        sys.exit(1)
+    instance = ec2.describe_instances(InstanceIds=[master_id]).get("Reservations")[0].get("Instances")[0]
+    ip_address = instance.get("PublicIpAddress")
+    if ip_address is None:
+        ip_address = instance.get("PrivateIpAddress")
+    state = instance.get("State").get("Name")
+    if state != "running" or ip_address is None:
+        LOGGER.info("MasterServer: %s\nCannot get ip address." % state.upper)
```

Comment:
upper is missing the `()`

Also you can use lazy string formatting in LOGGER by simply adding the parameters to replace as arguments:
`LOGGER.info("MasterServer: %s\nCannot get ip address.", state.upper)`

Reply:
fixed and changed 👍🏻